client/src/client-only-routes/ShowUser.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
```

Comment:
```suggestion
import { bindActionCreators } from 'redux';
import { navigate } from '@reach/router';
```
https://github.com/freeCodeCamp/freeCodeCamp/pull/36926#issuecomment-537857201